docs/modules/clusters/pages/ucn-dynamic-config.adoc

```diff
@@ -0,0 +1,43 @@
+= Dynamic Configuration API
+:description: You can use the dynamic configuration API to deploy resources from your client. This means that you can configure namespaces and their resources, and replace the resources at runtime for use when a user customization is next instantiated or executed.
```

Comment:
> to deploy resources from your client.

The dynamic configuration API is also available for a member `HazelcastInstance`, not just clients.